website/docs/docs/cloud/git/setup-azure.md

```diff
@@ -69,31 +69,31 @@ You also need to add another redirect URI to your Azure AD application. This red
 
 
 ## Connect Azure DevOps to your new app
-If you have already connected your Azure DevOps account to Active Directory, then you can proceed to [Connecting a service user](#connecting-a-service-user). However, if you're just getting set up, connect Azure DevOps to the Active Directory App you just created:
+If you have already connected your Azure DevOps account to Microsoft Entra ID, then you can proceed to [Connecting a service user](#connecting-a-service-user). However, if you're just getting set up, connect Azure DevOps to the Microsoft Entra ID App you just created:
 
 1. From your Azure DevOps account, select **Organization settings** in the bottom left.
-2. Navigate to Azure Active Directory.
+2. Navigate to Microsoft Entra ID.
 3. Click **Connect directory**.
 4. Select the directory you want to connect.
 5. Click **Connect**.
 
 <Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/connect AD to Azure DevOps.gif" title="Connecting Azure DevOps and Active Directory"/>
```

Comment:
```suggestion
<Lightbox src="/img/docs/dbt-cloud/connecting-azure-devops/connect AD to Azure DevOps.gif" title="Connecting Azure DevOps and Microsoft Entra ID"/>
```

Reply:
logging for posterity: chatted with docs team about the two options (mismatched caption/image vs still referencing AAD). as a group, we leaned towards first option